BM6toDB/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
Has this (BM6toDB) been tested in linux and windows?

Reply:
No. I haven't used it, and I don't know much about its use.